Percolator/Percolator.toolbox/.project

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>Percolator</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>toolbox.builder.TLAParserBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>toolbox.builder.PCalAlgorithmSearchingBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>toolbox.natures.TLANature</nature>
+	</natures>
+	<linkedResources>
+		<link>
+			<name>FiniteSets.tla</name>
+			<type>1</type>
+			<location>/usr/local/lib/tlaps/FiniteSets.tla</location>
```

Comment:
it is an assumption that all users install tlaps here. 
Why sending TLA toolbox project configuration too? I don't think it is fine here

Reply:
Double-checked only `Percolator___PercolatorModel.launch` is necessary when importing code into TLA+ toolbox, so I only retain that file.